src/client.cpp

```diff
@@ -166,7 +167,7 @@ void Client::window_unfocus_last() {
     if (lastfocus) {
         /* only emit the hook if the focus *really* changes */
         hook_emit_list("focus_changed", "0x0", "", nullptr);
-        ewmh_update_active_window(None);
+        Ewmh::get().updateActiveWindow(None);
```

Comment:
Is there any reason why it's `Ewmh::get().` here but `ewmh.` in the other places in this file?

Reply:
Because this function is static, whereas the other places are in non-static functions where we have the `ewmh` member available.

Reply:
Ah, right! That was my first thought, too, but apparently I forgot that the `static` would not be here but in the header, and then I assumed it to be non-static ;-)